api/tests/routes/pro/get_venues_of_offerer_from_siret.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+from pcapi.core.testing import assert_num_queries
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    def test_get_venues_of_offerer_from_siret(self, client):
+        siren = "123123123"
+        offerer = offerers_factories.OffererFactory(siren=siren)
+        user_offerer = offerers_factories.UserOffererFactory(user__email="user.pro@test.com", offerer=offerer)
+
+        permanent_venue = offerers_factories.VenueFactory(
+            name="Permanent", managingOfferer=user_offerer.offerer, siret=f"{siren}12323", isPermanent=True
+        )
+
+        offerers_factories.VenueFactory(
+            name="Not Permanent",
+            managingOfferer=user_offerer.offerer,
+            isPermanent=False,
+            siret=f"{siren}12324",
+        )
+
+        auth_request = client.with_session_auth(email=user_offerer.user.email)
+
+        with assert_num_queries(4):
```

Comment:
On peut au passage documenter les requêtes effectuées:
```suggestion
        with assert_num_queries(
            testing.AUTHENTICATION_QUERIES
            + 1  # retrieve Offerer
            + 1  # retrieve Venues
        ):
```
Pour vérifier qu'il n'y a pas de n+1, pourrais-tu ajouter une autre `Venue` permanente, ce qui déclencher le même nombre de requêtes ?
Au passage, on pourrait définir un `publicName` sur ce 2e lieu et vérifier l'ordre de la liste (triée par `common_name`).